src/Microsoft.VisualStudio.Web.CodeGeneration.Utils/project.json

```diff
@@ -32,5 +33,14 @@
         "System.Threading.Tasks.Parallel": "4.0.1-*"
       }
     }
+  },
+  "packOptions": {
+    "tags": [
+      "aspnetcore",
+      "aspnetcoremvc",
```

Comment:
(Probably more than just this package - I think a few others were not MVC-specific.)